dnf-testing.sh

```diff
@@ -110,8 +110,10 @@ run()
     [ ${#TESTS[@]} -eq 0 ] && TESTS=("${FEATURES[@]}")
     local failed=0
     for feature in "${TESTS[@]}"; do
-        sudo docker run --rm "$IMAGE" launch-test "$feature" dnf >&2 || :
-        [ $? -ne 0 ] && let ++failed
+        for command in dnf-2 dnf-3; do
```

Comment:
@ignatenkobrain This is local shell script - it runs test one by one. The behavior was not changed, only number of test is extended. Parallel run could be introduced but not by the PR.